### ISSUE TITLE
Quote style missing from `font-family-name-quotes` messages

### ISSUE THREAD

**Issue by ntwb** [stylelint/stylelint#1459]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

A bug where the quote style required is not included in the error message

> Which rule, if any, is this issue related to?

`font-family-name-quotes`

> What CSS is needed to reproduce this issue?

```css
.foo {
	font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "font-family-name-quotes": "always-where-recommended",
    "string-quotes": "double",
  }
}
```

> Which version of stylelint are you using?

 `6.6.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

CLI with `stylelint src/test.css --config .stylelintrc `"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

No

> What did you expect to happen?

Expected the message to include `double` as the expected quote style:

```shell
  2:50   ✖  Expected double quotes around font-family name "Segoe UI"        font-family-name-quotes  
```

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:"

```shell
  2:50   ✖  Expected  quotes around font-family name "Segoe UI"        font-family-name-quotes  
```

Note, there are two spaces between these two words `Expected  quotes`

**Comment by jeddy3** [comment]
I think this is working as intended (albeit with an extra space before the word "quotes").

These contextual quote rules now test for the presence of any type of quotes - they aren't aware if they should be checking for double or single. Checking if a quote should be double or single is the sole responsibility of `string-quotes`. This change was made to avoid redundancy and conflicts.

The extra space is fixed in `v7` as the rule was refactored.

**Comment by ntwb** [comment]
@jeddy3 Right you are, I'd had a quick look at `master` source and not the updates in `v7` 👍

**Comment by jeddy3** [comment]
Can we keep the "resolution: *" labels for just PRs please?

**Comment by ntwb** [comment]
> Can we keep the "resolution: *" labels for just PRs please?

I'll try to, high chance I'll forget a few more times, it's a long standing habit I've got with VCS software in general, not just GitHub hence if an issue/ticket or patch/pull request if it is invalid I'll mark it as invalid, same deal for wontfix, but I'll still try 😏
